packages/vscode-extension/package.nls.json

```diff
@@ -498,7 +491,7 @@
   "teamstoolkit.walkthroughs.steps.teamsToolkitExploreMore.title": "What's next?",
   "teamstoolkit.walkthroughs.steps.teamsToolkitPreview.description": "Press [F5](command:fx-extension.selectAndDebug?%5B%22WalkThrough%22%5D) or discover [Run and Debug](command:workbench.view.debug) panel on the activity bar, and click the play icon to locally preview your app in Teams context.\n[Run Local Preview (F5)](command:fx-extension.selectAndDebug?%5B%22WalkThrough%22%5D)\n__Tip: To run local preview, sign in to Microsoft 365 (organizational account) with custom app upload option.__",
   "teamstoolkit.walkthroughs.steps.teamsToolkitPreview.title": "Preview your Teams app locally",
-  "teamstoolkit.walkthroughs.title": "Get Started with Teams Toolkit",
+  "teamstoolkit.walkthroughs.title": "Build a Notification Bot",
```

Comment:
This string is also referenced in package.json, please help to check the change of it.